internal/contributebot/process_test.go

```diff
@@ -49,6 +55,31 @@ func TestProcessIssueEvent(t *testing.T) {
 				RemoveLabels: []string{"in progress"},
 			},
 		},
+		// Check issue title looks like "foo: bar".
+		{
+			description: "open with invalid issue title -> add comment",
+			action:      "opened",
+			title:       "foo",
+			want: &issueEdits{
+				AddComments: []string{issueTitleComment},
+			},
+		},
+		{
+			description:     "edit on invalid issue title but title didn't change -> no change",
+			action:          "edited",
+			title:           "foo",
+			prevTitle: "foo",
+			want:        &issueEdits{},
+		},
+		{
+			description:     "edit to invalid issue title -> add comment",
+			action:          "edited",
+			title:           "prev",
+			prevTitle: "foo",
+			want: &issueEdits{
+				AddComments: []string{issueTitleComment},
+			},
+		},
```

Comment:
Also test adding a comment not triggering the comment for the invalid issue title.

Reply:
Sorry, isn't that the 2nd testcase I added? (edit on an invalid issue title, but the title didn't change)? That's what "adding a comment" would look like.

Reply:
Ah, I didn't understand that. Perfect.